turbo.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**", "!.next/cache/**"]
+    }
+  }
```

Comment:
Can you add `globalDependencies` with `.env.` and `tsconfig.json` just like in https://turbo.build/repo/docs/reference/configuration#globaldependencies

Reply:
As well as the `globalDotEnv` https://turbo.build/repo/docs/reference/configuration#globaldotenv

Reply:
why we don't have `.env` ?

Reply:
Right now, yes, but it's pretty foreseeable to use `.env` in the future.

Reply:
Your are smarter than me 😁